plugins/UsersManager/API.php

```diff
@@ -336,15 +336,18 @@ public function getUsersPlusRole($idSite, $limit = null, $offset = 0, $filter_se
     /**
      * Returns the list of all the users
      *
-     * @param string $userLogins Comma separated list of users to select. If not specified, will return all users
+     * @param string|array $userLogins Comma separated list or array of users to select. If not specified, will return all users
      * @return array the list of all the users
      */
     public function getUsers($userLogins = '')
     {
         Piwik::checkUserHasSomeAdminAccess();
 
         $logins = array();
-        if (!empty($userLogins)) {
+
+        if (is_array($userLogins)) {
```

Comment:
or should we just fail if userLogins is not a string? In theory you may be able to send now some weird array structures to maybe cause an error? (haven't tested, just a thought)

Reply:
don't have a strong opinion on that. failing if no string is given works as well for sure

Reply:
lets do this to keep things simple 👍